src/hyperion/parameters/constants.py

```diff
@@ -3,6 +3,7 @@
 SIM_BEAMLINE = "BL03S"
 SIM_INSERTION_PREFIX = "SR03S"
 ISPYB_PLAN_NAME = "ispyb_readings"
```

Comment:
Nit: I would like it if these names reflected the plans more closely - e.g. `ISPYB_PREPLAN_READ` and 'ISPYB_MIDPLAN_READ' or something like that, to show that the two are related